boards/arm/colibri_imx7d_m4/pinmux.c

```diff
@@ -11,6 +11,21 @@ static int colibri_imx7d_m4_pinmux_init(struct device *dev)
 {
 	ARG_UNUSED(dev);
 
+#ifdef CONFIG_GPIO_IMX_PORT_1
+	/* GPIO1_IO02 Mux Config */
+	*(&IOMUXC_LPSR_SW_MUX_CTL_PAD_GPIO1_IO02) = 0;
+	*(&IOMUXC_LPSR_SW_PAD_CTL_PAD_GPIO1_IO02) = 0;
```

Comment:
Why are you getting a reference and immediately dereferencing it? This seems unnecessary